test/acceptance_with_go_client/object_property_tests/batch_test.go

```diff
@@ -40,7 +40,7 @@ func TestObjectProperty_Batch(t *testing.T) {
 		},
 		{
 			name:   "grpc api",
-			config: wvt.Config{Scheme: "http", Host: "localhost:8080", GrpcConfig: grpc.Config{Enabled: true, Host: "localhost:50051"}},
```

Comment:
Expected change ?

Reply:
Yes, bc I have update go client to official `v4.11.0` version